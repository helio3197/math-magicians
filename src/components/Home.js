import React from 'react';

const Home = () => (
  <div
    style={{
      padding: '24px calc(6px + 5vw)',
      height: '100%',
    }}
  >
    <h2>Welcome to our page!</h2>
    <p>
      This is a simple React-built website that let you perform simple math operations through
      our calculator app.
    </p>
    <p>
      Also, there is a math quote in our quote section so you can get inspired.
    </p>
  </div>
);

export default Home;
