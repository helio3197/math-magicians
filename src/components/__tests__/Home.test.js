import { render } from '@testing-library/react';
import Home from '../Home';

describe('Test for Home component', () => {
  test('The component should render', () => {
    const { asFragment } = render(<Home />);

    expect(asFragment()).toMatchSnapshot();
  });
});
