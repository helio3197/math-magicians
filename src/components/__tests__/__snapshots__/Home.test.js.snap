// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Home component The component should render 1`] = `
<DocumentFragment>
  <div
    style="height: 100%;"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      This is a simple React-built website that let you perform simple math operations through our calculator app.
    </p>
    <p>
      Also, there is a math quote in our quote section so you can get inspired.
    </p>
  </div>
</DocumentFragment>
`;
